data/json/furniture_and_terrain/furniture-fireplaces.json

```
@@ -69,6 +69,33 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_makeshift_brazier",
+    "name": "makeshift brazier",
+    "looks_like": "bucket",
+    "description": "A metal bucket with holes, in which you can safely start a fire.  It's too small to fit any fuel item larger than some splintered wood.",
+    "symbol": "#",
+    "color": "red",
+    "move_cost_mod": 2,
+    "coverage": 35,
+    "required_str": 4,
```

There's no volume limit, like the description suggests - this will add one.
```suggestion
    "required_str": 4,
    "max_volume": "5 L",
```

---

I thought `pocket-data` would limit the volume, apparently not. Thanks for the heads up.